src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -829,12 +1155,13 @@ uint8_t P036_data_struct::display_scroll(ePageScrollSpeed lscrollspeed, int lTas
         log += F(" CharsRemoved: "); log += iCharToRemove;
         addLog(LOG_LEVEL_INFO, log);
         log.clear();
-        log += F(" -> Changed to: "); log += ScrollingPages.LineOut[j];
-        log += F(" Length: "); log += ScrollingPages.LineOut[j].length();
-        log += F(" PixLength: "); log += display->getStringWidth(ScrollingPages.LineOut[j]);
+        log += F(" -> Changed to: "); log += ScrollingPages.Out[j].SPLcontent;
+        log += F(" Length: "); log += ScrollingPages.Out[j].SPLcontent.length();
+        display->setFont(FontSizes[LineSettings[ScrollingPages.Out[j].SPLidx].fontIdx].fontData);
```

Comment:
Idem, performing actions depending on the set log level.

Reply:
It's not an error.
display->setFont() must be used to calculate the display->getStringWidth() according to the individual font.
The StringWidth should just be logged, nothing else.
Later the setting of display->setFont() is overwritten by the correct values for this line.